components/public-api-server/pkg/apiv1/user.go

```diff
@@ -67,6 +67,24 @@ func (s *UserService) ListSSHKeys(ctx context.Context, req *connect.Request[v1.L
 	}), nil
 }
 
+func (s *UserService) GetGitToken(ctx context.Context, req *connect.Request[v1.GetGitTokenRequest]) (*connect.Response[v1.GetGitTokenResponse], error) {
```

Comment:
I think eventually this shouldn't belong on the UserService, but doesn't matter at this stage, in the experimental package.